### DISCUSSION TITLE
Save data locally on RaspberryPi

### DISCUSSION THREAD

**Discussion by StefanThuwis** [ketilmo/balena-ads-b#83] [General]
Hi,

First of all, thank you for the repository, it made my life easy to stream data to the different ADS-B networks.

I am trying to analyse my data for a university paper, but I have difficulties finding where or how I can save the data locally. Ideally, I would like to save the data directly on my memory card in the raspberryPi. Is this possible?

Stef

**Comment by ketilmo** [comment]
Hi @StefanThuwis,

Thank you for your feedback. It's appreciated.

To ensure the stability and durability of the feeding device (specifically, its SD card), balena-ads-b stores as little data as possible locally. This is by design. If you need to store your data, you could for instance setup a small server that pulls data from one of the data ports on balena-ads-b. 

An easier alternative: Given that you feed your data to OpenSky Network and are doing a university paper, you can apply for data access to the OpenSky Network data API. I would suggest trying this alternative first.

**Read more here:**
https://opensky-network.org/data/apply?view=dataapplicationform
https://opensky-network.org/data/impala
https://openskynetwork.github.io/opensky-api/

Let us know which route you choose.
